avocado/core/settings.py

```diff
@@ -164,34 +167,36 @@ def __init__(self, config_path=None):
             _config_dir_system = os.path.join(cfg_dir, 'avocado')
             _config_dir_system_extra = os.path.join(cfg_dir, 'avocado', 'conf.d')
             _config_dir_local = os.path.join(user_dir, '.config', 'avocado')
-            _config_path_intree = os.path.join(os.path.abspath(_source_tree_root),
-                                               'avocado', 'etc', 'avocado')
-            _config_path_intree_extra = os.path.join(_config_path_intree, 'conf.d')
 
             config_filename = 'avocado.conf'
             config_path_system = os.path.join(_config_dir_system, config_filename)
             config_path_local = os.path.join(_config_dir_local, config_filename)
-            config_path_intree = os.path.join(_config_path_intree, config_filename)
 
             config_system = os.path.exists(config_path_system)
             config_system_extra = os.path.exists(_config_dir_system_extra)
             config_local = os.path.exists(config_path_local)
-            config_intree = os.path.exists(config_path_intree)
-            config_intree_extra = os.path.exists(_config_path_intree_extra)
-            config_pkg_base = os.path.join('etc', config_filename)
+            config_pkg_base = os.path.join('etc', 'avocado', config_filename)
```

Comment:
I was trying to make sense of this fix, and it looks like it's a fix (the right location is indeed `etc/avocado/avocado.conf`).